ruby_connectors/spec/github_connector/cataloger_spec.rb

```
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "ruby_connectors/github_connector/cataloger"
+
+RSpec.describe RubyConnectors::GithubConnector::Cataloger do
```

rubocop

[convention] Spec path should end with `ruby_connectors/github_connector/cataloger*_spec.rb`.

(RSpec/SpecFilePathFormat)